crates/module-info-parser/src/sdk_version/version_embedder.rs

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use crate::ModuleInfoResult;
+use crate::ModuleInfoError;
+use fluence_sdk_main::VERSION_SECTION_NAME;
+
+use walrus::ModuleConfig;
+use walrus::CustomSection;
+use walrus::IdsToIndices;
+
+use std::path::Path;
+use std::borrow::Cow;
+
+#[derive(Debug, Clone)]
+pub(super) struct VersionCustomSection(Vec<u8>);
```

Comment:
Why `Vec<u8>` here? You can convert it inside the `data` function. Version is always just a string it seems, so I think it is better to express that in types

Reply:
Yep, good idea